test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

```diff
@@ -385,40 +385,41 @@ describe('Java expression language (JEXL) based transformations plugin', functio
         });
     });
 
-    describe('When an update comes for attributes with string expression and type integer', function () {
```

Comment:
We have also a variant of this tests for ngsiv2

test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

That file doesn't need modifications? (looking to CI results, it seems it doesn't, but I wonder why...)